arxiv/base/config.py

```diff
@@ -1,46 +1,51 @@
 """Flask configuration."""
 
+from typing import Optional
 import os
+from urllib.parse import urlparse
 
 SERVER_NAME = None
 
-EXTERNAL_URLS = [
-    ("twitter", os.environ.get("ARXIV_TWITTER_URL",
-                               "https://twitter.com/arxiv")),
-    ("blog", os.environ.get("ARXIV_BLOG_URL",
-                            "https://blogs.cornell.edu/arxiv/")),
-    ("wiki", os.environ.get("ARXIV_WIKI_URL",
-                            "https://confluence.cornell.edu/display/arxivpub/"
-                            "arXiv+Public+Wiki")),
-    ("accessibility", os.environ.get("ARXIV_ACCESSIBILITY_URL",
-                                     "mailto:web-accessibility@cornell.edu")),
-    ("library", os.environ.get("ARXIV_LIBRARY_URL",
-                               "https://library.cornell.edu")),
-    ("acknowledgment", os.environ.get(
-        "ARXIV_ACKNOWLEDGEMENT_URL",
-        "https://confluence.cornell.edu/x/ALlRF"
-    )),
-]
-"""External URLs, configurable via environment variables."""
+A11Y_URL = os.environ.get("ARXIV_ACCESSIBILITY_URL",
+                          "mailto:web-accessibility@cornell.edu")
 
-ARXIV_URLS = [
-    ("help", "/help"),
-    ("contact", "/help/contact"),
-    ("search_box", "/search"),
-    ("search_advanced", "/search/advanced"),
-    ("account", "/user"),
-    ("login", "/user/login"),
-    ("logout", "/user/logout"),
-    ("home", "/"),
-    ("pdf", "/pdf/<arxiv:paper_id>"),
+URLS = [
```

Comment:
It seems like we'll need to have something like:
BASE_SERVER='arxiv.org'
because in many cases it will need to be configured:
BASE_SERVER='beta.arxiv.org'